### PR TITLE
importing ab2d-env rds

### PR DESCRIPTION
**Pull request by knavapbc**
## 🎫 Ticket

https://jira.cms.gov/browse/PLT-562=3

## 🛠 Changes

no change expected

## ℹ️ Context

Changes on the .tf files to make sure after import, terrafor plan do not show any breaking changes

## 🧪 Validation
comment will show the output of terraform plan

### CODE REVIEW

**Comment by knavapbc** [comment]
ab2d dev
```
Terraform will perform the following actions:

  # aws_db_instance.api will be updated in-place
  ~ resource "aws_db_instance" "api" {
      + apply_immediately                     = false
        id                                    = "db-2I2FGYBMWFOUXVTVAN2BNHD2HM"
        tags                                  = {
            "Name"             = "ab2d-dev-rds"
            "contact"          = "ab2d-ops@semanticbits.com"
            "cpm backup"       = "Monthly"
            "environment"      = "ab2d-dev"
            "role"             = "db"
            "terraform_module" = "data"
        }
      ~ tags_all                              = {
          + "application"      = "ab2d"
          + "business"         = "oeda"
          + "code"             = "https://github.com/CMSgov/ab2d-bcda-dpc-platform/tree/main/terraform/services/api-rds"
          + "component"        = "api-rds"
            # (7 unchanged elements hidden)
        }
        # (52 unchanged attributes hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.

```

**Comment by knavapbc** [comment]
sbx
```
Terraform will perform the following actions:

  # aws_db_instance.api will be updated in-place
  ~ resource "aws_db_instance" "api" {
      + apply_immediately                     = false
        id                                    = "db-5DGZCD7F4EKHI554AZUVKBPNNQ"
        tags                                  = {
            "Name"             = "ab2d-sbx-sandbox-rds"
            "contact"          = "ab2d-ops@semanticbits.com"
            "cpm backup"       = "Monthly"
            "environment"      = "ab2d-sbx-sandbox"
            "role"             = "db"
            "terraform_module" = "data"
        }
      ~ tags_all                              = {
          + "application"      = "ab2d"
          + "business"         = "oeda"
          + "code"             = "https://github.com/CMSgov/ab2d-bcda-dpc-platform/tree/main/terraform/services/api-rds"
          + "component"        = "api-rds"
            # (7 unchanged elements hidden)
        }
        # (52 unchanged attributes hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.


```

**Comment by knavapbc** [comment]
prod
```

  # aws_db_instance.api will be updated in-place
  ~ resource "aws_db_instance" "api" {
      + apply_immediately                     = false
        id                                    = "db-PZYY2TK3ZJ3HODBJPUZ5R4646E"
        tags                                  = {
            "Name"             = "ab2d-east-prod-rds"
            "contact"          = "ab2d-ops@semanticbits.com"
            "cpm backup"       = "Monthly"
            "environment"      = "ab2d-east-prod"
            "role"             = "db"
            "terraform_module" = "data"
        }
      ~ tags_all                              = {
          + "application"      = "ab2d"
          + "business"         = "oeda"
          + "code"             = "https://github.com/CMSgov/ab2d-bcda-dpc-platform/tree/main/terraform/services/api-rds"
          + "component"        = "api-rds"
            # (7 unchanged elements hidden)
        }
        # (52 unchanged attributes hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.


```

**Review by gsf — CHANGES_REQUESTED** [review]
Please add the check-terraform-fmt job back into the api-rds-plan workflow and confirm successful plan via that check.

File: .github/workflows/api-rds-plan.yml [hunk 1]
```diff
@@ -7,6 +7,13 @@ on:
...
+  check-terraform-fmt:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./actions/setup-tfenv-terraform
+      - run: terraform fmt -check -diff -recursive terraform/services/api-rds
+
   terraform-plan:
     needs: check-terraform-fmt
     permissions:
```
Comment (gsf) [inline .github/workflows/api-rds-plan.yml:19]: AB2D also includes a "prod-test" environment with a database. The matrix below will need to be extended with the environment.

File: terraform/services/api-rds/main.tf [hunk 2]
```diff
@@ -1,10 +1,48 @@
+locals {
+  db_name = {
+    ab2d = {
+      dev  = "ab2d-dev"
+      test = "ab2d-east-prod-test"
```
Comment (gsf) [inline terraform/services/api-rds/main.tf:5]: The ab2d test environment database is "ab2d-east-impl". See my comment above for handling of the ab2d-east-prod-test database.

**Pull request merged**